### DISCUSSION TITLE
After update wireui Uncaught SyntaxError: Unexpected end of input

### DISCUSSION THREAD

**Discussion by jasha1974** [wireui/wireui#268] [Q&A]
... and lot of errors :

Uncaught SyntaxError: Unexpected end of input (at scripts?id=9d225009069d65aea9807fdaf173cfe7:1:97398)
http://127.0.0.1:8000/wireui/assets/scripts?id=9d225009069d65aea9807fdaf173cfe7

Uncaught ReferenceError: wireui_dialog is not defined
    at eval (eval at safeAsyncFunction (app.js:485:14), <anonymous>:3:16)
    at app.js:503:21
    at tryCatch (app.js:423:12)
    at evaluate (app.js:448:32)
    at app.js:2599:15
    at Function.<anonymous> (app.js:1166:55)
    at flushHandlers (app.js:560:46)
    at stopDeferring (app.js:565:5)
    at deferHandlingDirectives (app.js:568:3)
    at initTree (app.js:778:3)

etc. any help ? thans a lot

**Comment by jasha1974** [comment]
"require": {
        "php": "^8.1",
        "fruitcake/laravel-cors": "^2.0",
        "guzzlehttp/guzzle": "^7.0.1",
        "laravel/framework": "^9.0",
        "laravel/sanctum": "^2.11",
        "laravel/tinker": "^2.5",
        "livewire/livewire": "^2.10.5",
        "maatwebsite/excel": "^3.1.40",
        "power-components/livewire-powergrid": "^1.5.1",
        "rappasoft/laravel-livewire-tables": "^1.22",
        "wire-elements/modal": "^1.0.6",
        "wireui/wireui": "^v1.1.3"
    },

**Comment by ph7jack** [comment]
@jasha1974 Could you add more context?
